lib/utils/configloader.js

```diff
@@ -45,6 +45,7 @@ class ConfigLoader {
                 this.config.SENSITIVE_HEADERS = body.sensitiveHeaders;
                 this.config.ACTIVITIES_TIMEOUT = body.connectTimeout;
                 this.config.API_TIMEOUT_MS = body.riskTimeout;
+                this.config.MODULE_MODE = body.moduleMode === "blocking" ? 1 : 0;
```

Comment:
Would be nicer if you use the enum you created for that, like:
`this.config.MODULE_MODE = body.moduleMode === "blocking" ? this.config.MONITOR_MODE.BLOCK : this.config.MONITOR_MODE. MONITOR;`